Sources/NIO/ByteBuffer-aux.swift

```diff
@@ -373,6 +373,21 @@ extension ByteBuffer {
         self._moveWriterIndex(forwardBy: written)
         return written
     }
+    
+    /// Writes `byte` `repeated` times. Moves the writer index forward by the number of bytes written.
+    ///
+    /// - parameter byte: The `UInt8` byte to repeat.
+    /// - parameter count: How many times to repeat the given `byte`
+    /// - returns: How many bytes were written.
+    @discardableResult
+    @inlinable
```

Comment:
can we take off this `@inlinable`? The parameters here are `UInt8` and `Int`, if anything, inlining this would make it worse.